lib/rspec/core/configuration.rb

```diff
@@ -484,6 +484,11 @@ def reset
         @formatter_loader = nil
       end
 
+      def reset_reporter
```

Comment:
This either needs some docs or a `# @private` to satisfy our documentation coverage check :)